internal/provider/svm_resource.go

```diff
@@ -373,5 +427,16 @@ func (r *SvmResource) Delete(ctx context.Context, req resource.DeleteRequest, re
 
 // ImportState imports a resource using ID from terraform import command by calling the Read method.
 func (r *SvmResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+	idParts := strings.Split(req.ID, ",")
+
+	if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+		resp.Diagnostics.AddError(
+			"Unexpected Import Identifier",
+			fmt.Sprintf("Expected import identifier with format: attr_one,attr_two. Got: %q", req.ID),
```

Comment:
Something @chuyich metioned in the other review that i linked. We should list the attribute here.